### DISCUSSION TITLE
GitHub Action for Metrics running for 3 hours (Sometimes)

### DISCUSSION THREAD

**Discussion by JayantGoel001** [lowlighter/metrics#671] [Q&A] [answered]
Hi @lowlighter,
Sometimes Github Action keeps on running for hours.
https://github.com/JayantGoel001/JayantGoel001/actions/runs/1483824029
https://github.com/JayantGoel001/JayantGoel001/actions/runs/1475041005

Can you kindly help me with this issue?

**Comment by lowlighter** [comment, marked as answer]
Hi!
This is difficult to debug because there are no logs displayed in the workflow runs.
If you achieve to find out which plugin/option is causing such delays I may be able to help you on that matter 🙂

**Comment by JayantGoel001** [comment]
> Hi! This is difficult to debug because there are no logs displayed in the workflow runs. If you achieve to find out which plugin/option is causing such delays I may be able to help you on that matter 🙂

Well, I made two changes
1. merged the latest changes(https://github.com/JayantGoel001/metrics/commit/90146cd72e40bf17a387b33d9b47ae1f03569aaf)
2. I removed plugin_people_user which was showing **Unexpected input(s) 'plugin_people_user'** (https://github.com/JayantGoel001/JayantGoel001/actions/runs/1485012721)

And it is working now.😁

**Reply by lowlighter** [reply]
Awesome!
